docs/courses/batch_runs_bulletin_board_parallelization.rst

```diff
@@ -0,0 +1,240 @@
+######
+
+- :download:`initbatpar.py <code/initbatpar.py>`
+- :ref:`code walkthrough <initbatpar_walkthrough>`
+
+Things to do
+------------
+
+1. Run a few simulations with initonerun.py just to see how the cell responds to injected current. You might try to find the smallest and largest stimuli that elicit repetitive dendritic spiking.
+
+2. 
+
+    Compare results produced by serial and parallel simulations, to verify that parallelization hasn't broken anything. For example (on Linux and macOS):
+
+    .. code-block::
+
+        python initbatser.py
+        mv fi.dat fiser.dat
+        python initbatpar.py
+        mv fi.dat finompi.dat
+        mpiexec -n 4 python initbatpar.py
+        mv fi.dat fimpi4.dat
+        cmp fiser.dat finompi.dat
+        cmp fiser.dat fimpi4.dat
+    
+    Instead of ``cmp``, MSWin users will have to use ``fc`` in a "Command Prompt" (cmd) or Power Shell window.
+
+3. 
+
+    Evaluate and compare performance of the serial and parallel programs.
+    Here are results of some tests I ran:
+
+    .. code-block::
+
+        NEURON 7.5 (266b5a0) 2017-05-22 under Windows Subsystem for Linux
+        on a quad core desktop.
+        initbatser     6.16 s
+        initbatpar
+        without MPI  6.03
+        with MPI            speedup
+        n = 1        6.03   1 (performance baseline)
+            2        3.41   1.77
+            3        2.62   2.30
+            4        2.05   2.94
+
+4.
+    Make a copy of initbatpar.py and edit it, inserting print calls that reveal the sequence of execution, i.e. which processor is doing what. These statements should report whatever you think would help you understand program flow. Here are some suggestions for things you might want to report:
+
+    - the identity of the host process (i.e. the ``pc.id``)
+    - the name of the proc or func that is being entered or exited
+    - the value that is being passed or returned
+    - the index of the simulation that is being run
+    - anything else that you think might illuminate this dark little corner of computational neuroscience
+
+    Focus on the parts of the program that are involved in the master's principal function (posting jobs and gathering results), and the workers' principal function (entering, executing, and exiting ``fi``).
+
+    After inserting the print calls, change NRUNS to 3 or 4, then run a serial simulation and see what happens.
+
+    Next run parallel simulations with -n 1, 2, 3 or 4 and see what happens. Do the monitor reports make sense?
+
+5. 
+    Examine an f-i curve from data saved to one of the dat files.
+
+    .. code-block::
+    
+        python -i initplotfi.py
+
+    then use its file browser to select one of the dat files.
+
+    Examine :download:`initplotfi.py <code/initplotfi.py>` to see how it takes advantage of procs that are built into NEURON's standard run library (UNIX/Linux users see :file:`nrn/share/nrn/lib/hoc/stdlib.hoc`, MSWin users see :file:`c:\\nrn\\lib\\hoc\\stdlib.hoc`).
```

Comment:
I'm not sure it's obvious where to find the installed copy of stdlib.hoc, it will depends on the build. Why not just link to github?